### PR TITLE
OPENEUROPA-1116: Import and create taxonomies needed for the demo

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: robo.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+data:
+  - name: "corporate-body"
+    graph: "http://publications.europa.eu/resource/authority/corporate-body"
+    url: "http://publications.europa.eu/resource/cellar/07e1a665-2b56-11e7-9412-01aa75ed71a1.0001.10/DOC_1"
+
+  - name: "target-audience"
+    graph: "http://publications.europa.eu/resource/authority/target-audience"
+    url: "http://publications.europa.eu/resource/cellar/4cb35e04-75c5-11e7-b2f2-01aa75ed71a1.0001.08/DOC_1"
+
+  - name: "organization-audience"
```
Comment (upchuk) [inline robo.yml]: `organization-type` no?

**Review by upchuk — CHANGES_REQUESTED** [review]
Quick typo I think

File: robo.yml (hunk 1 above)
Comment (ademarco) [reply robo.yml]: yep!

**Review by upchuk — DISMISSED** [review]
This worked for me. Stuff got imported, stuff gets saved.

**Review by aritomelo — APPROVED** [review]
tested and it works fine!

**Review by brummbar — APPROVED** [review]
Nice, good to go.

**Pull request merged**